tools/virtualization/playing-around-with-various-vga-providers.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Playing around with various VGA providers  
+subsection: virtualization  
+order: 11  
+---
+
+# Playing around with various VGA providers
+
+> More information can be found here https://www.kraxel.org/blog/2019/09/display-devices-in-qemu/
```

Comment:
Please move this to `## References` at the bottom of the page and make it a hyperlink.

Reply:
Addressed here https://github.com/developer-portal/content/pull/394/commits/c8c7dbab93f7ac23cdbcb93b99cbebdb2da63799